ci-operator/jobs/stolostron/policy-collection/stolostron-policy-collection-main-presubmits.yaml

```diff
@@ -57,32 +57,32 @@ presubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
-    trigger: (?m)^/test( | .* )opp-ocp4.16-interop-images,?($|\s.*)
+    trigger: (?m)^/test( | .* )opp-ocp4.17-interop-images,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
Did you change this line on purpose?  I think this might be a change that causes problems in the policy-collection pull requests.

Reply:
No I did not, it seems has been changed after executing `make update`.
Let me change it back to `false` and rerun